front/components/assistant/manager/AssistantsTable.tsx

```diff
@@ -127,7 +144,7 @@ const getTableColumns = ({
         </DataTable.CellContent>
       ),
       meta: {
-        className: "w-32",
+        className: "hidden @sm:w-32 @sm:table-cell",
```

Comment:
I have no idea what I'm doing, unsure if it's the best way to do media queries. But reading sparkle story book it seems this is the way™

Reply:
so basically by default it's `display: hidden` (= we don't show anything) but when it's bigger than `sm` size it will be `display:table-cell` (= which how it's supposed to be for table cell ^^)